appium/tests/specs/mock/setup/UpdateOlderPubKeyToNewer.spec.ts

```diff
@@ -0,0 +1,209 @@
+import { MockApi } from 'api-mocks/mock';
+import {
+    SplashScreen,
+    SetupKeyScreen,
+    MailFolderScreen,
+    NewMessageScreen,
+    PublicKeyDetailsScreen
+} from '../../../screenobjects/all-screens';
+import {attesterPublicKeySamples} from "../../../../api-mocks/apis/attester/attester-endpoints";
+import MenuBarScreen from "../../../screenobjects/menu-bar.screen";
+import SettingsScreen from "../../../screenobjects/settings.screen";
+import ContactScreen from "../../../screenobjects/contacts.screen";
+import ContactPublicKeyScreen from "../../../screenobjects/contact-public-key.screen";
+import {CommonData} from "../../../data";
+import DataHelper from "../../../helpers/DataHelper";
+
+
+describe('SETUP: ', () => {
+
+  it('app updates older public keys to newer but not vice versa', async () => {
+
+      let firstFetchedDate, secondFetchedDate, thirdFetchedDate, fourthFetchedDate;
+
+      const mockApi = new MockApi();
+
+      mockApi.fesConfig = {
+          clientConfiguration: {
+              flags: ["NO_PRV_CREATE", "NO_PRV_BACKUP", "NO_ATTESTER_SUBMIT", "PRV_AUTOIMPORT_OR_AUTOGEN", "FORBID_STORING_PASS_PHRASE"],
+              key_manager_url: "https://ekm.flowcrypt.com",
+          }
+      };
+      mockApi.attesterConfig = {
+          servedPubkeys: {
+              'updating.key@example.test': attesterPublicKeySamples.keyOlderVersion
+          }
+      };
```

Comment:
Looks like indentation is a bit off on lines 22-36, they should be moved 1 tab to the left.

Reply:
changed